src/js/step.js

```diff
@@ -299,13 +259,13 @@ export class Step extends Evented {
   destroy() {
     if (isElement(this.el) && this.el.parentNode) {
       this.el.parentNode.removeChild(this.el);
-      delete this.el;
+      this.el = null;
```

Comment:
Why set to `null` instead of deleting?

Reply:
`delete` has the effect of setting things to `undefined`, whereas setting to `null` is us saying that we aren't concerned about the reference anymore. 

I [tend to favor the latter](https://medium.com/@arieldi/anyway-stop-setting-your-variables-to-undefined-difference-undefined-vs-null-25f15d907fb4), generally -- but it also felt more consistent with the way we're cleaning up in other parts of our code.

Reply:
I think the idea is it should be completely gone, which `delete` gives us. Suppose it's no big deal, but would be good if we minimized changes.

Reply:
Just updated to revert back to `delete`.